modular_skyrat/modules/cargo/code/items/AFAD.dm

```diff
@@ -0,0 +1,150 @@
+/obj/item/gun/afad
```

Comment:
Is this just a copy-paste of the medigun code, but with a different typepath, just because you want to have a different healing behavior? Because if so, that's definitely not the way to go.

Reply:
redone